Drone/drone.py

```diff
@@ -0,0 +1,18 @@
+import threading
+import logging
+
+# Ours
+from drone_base import DroneBase
+from ..Utilities import constants as c
+
+class Drone(DroneBase):
+    """
+    Concrete implementation of DroneBase. See drone__base.py for
+    documentation.
+    """
+
+    def loadDevices(self):
```

Comment:
should be load_devices. see https://www.python.org/dev/peps/pep-0008/ for official style guide
```suggestion
    def load_devices(self):
```